docs/docsite/rst/network/user_guide/network_debug_troubleshooting.rst

```diff
@@ -509,6 +509,8 @@ Suggestions to resolve:
 
   Suggestions to resolve:
 
+  Some modules support a ``timeout`` option.
+  
```

Comment:
tempted to add (which is different from the tasks's ``timeout`` keyword)

Reply:
@bcoca More detail never hurt the docs. How about "Some modules support a ``timeout`` option, which is different to the ``timeout`` keyword for tasks."